content/intuitionistic-logic/semantics/semantic-notions.tex

```diff
@@ -19,23 +19,46 @@
   that $\mSat{M}{\Gamma}[w]$, $\mSat{M}{!A}[w]$.
 \end{defn}
 
+$\Gamma \Entails !A$ is defined by quantifying over
+models as well as worlds $w \in W$. The next proposition tells us that 
+$\Gamma \Entails !A$ holds iff for every model~$\mModel{M}$ such
+  that $\mSat{M}{\Gamma}$, $\mSat{M}{!A}$.
+
 \begin{prop}\ollabel{prop:sat-entails}
   \begin{enumerate}
-  \item\ollabel{prop:sat-entails1} If $\mSat{M}{\Gamma}[w]$ and
-    $\Gamma \Entails !A$, then $\mSat{M}{!A}[w]$.
-  \item\ollabel{prop:sat-entails2} If $\mSat{M}{\Gamma}$ and $\Gamma
-    \Entails !A$, then $\mSat{M}{!A}$.
+  \item\ollabel{prop:sat-entails1} If $\Gamma
+    \Entails !A$ and $\mSat{M}{\Gamma}$, then $\mSat{M}{!A}$.
+  \item\ollabel{prop:sat-entails2} If for every model~$\mModel{M}$ such
```

Comment:
I think this deserves its own proposition. Also, the proof that M, w |=A iff M_w |= A should be put in the section on relational models, right after the proposition that satisfaction is monotonic (which it should use).

Reply:
@feffemannen if you agree you can either: revert this change, add another separate proposition with the content of (2), add the definition of M_w and proposition proof that a) M, v |= A iff M_w, v |= A (for Rwv) and b) M_w, w |=A iff M_w |=A to relational-models.tex which the proof of (2) then references. Or: you can just tell me what you think and if you agree, I'll make the change after I approve the PR.

Reply:
Yes, I think you're right. Maybe it's best you do both of these changes. Thanks.